### DISCUSSION TITLE
What is the main flavor of the alcoholic beverage Cointreau?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#743] [Galaxy Trivia] [answered]
**Bonus:** Describe a recipe to make a good cocktail with it.

**Comment by SANKARAMDAS** [comment]
"Cointreau is clear in both clarity, has crisp, smooth, and its intense orange aroma. It is full bodied and is sweet on the palate with a strong orange flavor. The alcohol strength helps to keep the orange liqueur from being cloying but the raw spirit peaks out at times. Very mild spices like cinnamon and nutmeg appear. The colorless liqueur measures 40 percent ABV, or 80 proof.

Simply delightful in sophisticated cocktails, such as the Cointreau Noir Spritz with fresh lemon juice and chilled Champagne, or the Cointreau Noir Bubbly Berry with blackberry purée, fresh lime juice and a dash of Brut Champagne.

**Comment by peterbaccaro** [comment, marked as answer]
Cointreau is a type of triple sec liqueur, which is a type of clear, sweet, orange-flavoured liqueur. The main flavour of Cointreau is **sweet and bitter orange**, which is derived from the peels of both sweet and sour oranges. 
Cointreau is **made using a combination of sweet and bitter orange peels, alcohol, and sugar**, and has a distinct citrusy aroma and flavour. It is often used in cocktails such as Margaritas, Cosmopolitans, and Sidecars, as its sweet orange flavour complements many different types of spirits.

Here's a simple **recipe for making a Margarita using Cointreau**:

**Ingredients:**
- 2 oz tequila
- 1 oz Cointreau
- 1 oz freshly squeezed lime juice
- Salt (for rimming the glass)
- Ice

**Instructions:**
- Rim a chilled cocktail glass with salt. To do this, rub a lime wedge around the rim of the glass and then dip it into a dish of salt.
- Fill a cocktail shaker with ice. Add the tequila, Cointreau, and lime juice.
- Shake vigorously for about 10-15 seconds until the cocktail shaker becomes frosty.
- Strain the cocktail into the salt-rimmed glass over ice.
- Garnish with a lime wedge if desired, and serve immediately.

**Reply by ParanoidUser** [reply]
Cheers! 🍹
